code/modules/mob/living/simple_animal/abnormality/he/singing_machine.dm

```diff
@@ -130,49 +161,96 @@ Finally, an abnormality that DOESN'T have to do any fancy movement shit. It's a
 	return ..()
 
 /mob/living/simple_animal/hostile/abnormality/singing_machine/ZeroQliphoth(mob/living/carbon/human/user) // WARNING: Don't call this on its own. Several zero-qliphoth behaviors rely on its qliphoth actually being 0.
-	icon_state = "singingmachine_open_[cleanliness]" // Machine opens and starts making horrible empty grinding noises.
-	icon_living = icon_state
-	update_icon()
 	playsound(src, 'sound/abnormalities/singingmachine/open.ogg', 200, 0, playRange)
+	machine_hinge.moveHinge(-90, 1.5 SECONDS, -2, 6)
+	particle_note_broken = new(get_turf(src))
+	particle_grind = new(get_turf(src))
 	grindNoise = new(list(src), TRUE)
-	playStatus = 1
+	playStatus = GRINDING
 	return
 
+/mob/living/simple_animal/hostile/abnormality/singing_machine/proc/noiseEffect()
```

Comment:
Please make all words in Procs start with an Uppercase letter. PascalCase is the preferred method despite contribute guidelines saying otherwise for the time being. ( EX: NoiseEffect() )

Reply:
argh, will do, i had read on the contributor guidelines recently, didnt know they were outdated.